syntax = "proto3";

package myservice;

service MyService {
  rpc MyMethod (MyRequest) returns (MyResponse);
}

message MyRequest {
  string message = 1;
}

message MyResponse {
  string message = 1;
}
